entry.go

```diff
@@ -241,6 +250,11 @@ func (entry Entry) log(level Level, msg string) {
 }
 
 func (entry *Entry) fireHooks() {
+	// add a check for logger being nil because entry is exposed
+	if entry.Logger == nil {
+		fmt.Fprintf(os.Stderr, "Logger not attached\n")
```

Comment:
I feel like this shouldn't happen. If no logger is attached, I don't expect any output.

Reply:
This is an error scenario. So at least the logs should go to the standard error output. No error should be blindly ignored.